collection-scripts/gather_nodes

```diff
@@ -17,20 +32,19 @@ oc create -f $CRD_MANIFEST
 oc adm policy add-scc-to-user privileged -n node-gather -z node-gather
 oc create -f $DAEMONSET_MANIFEST
 
-while true
-do
-    line=$(oc get ds node-gather-daemonset -o=custom-columns=DESIRED:.status.desiredNumberScheduled,READY:.status.numberReady --no-headers -n node-gather)
+COUNTER=0
+until check_node_gather_pods_ready || [ $COUNTER -eq 300 ]; do
+   (( COUNTER++ ))
+   sleep 1
+done
 
-    DESIRED=$(echo $line | awk -F ' ' '{print $1}')
-    READY=$(echo $line | awk -F ' ' '{print $2}')
-    if [[ $READY -eq $DESIRED ]]
-    then
-       break
-    fi
-    sleep 1
+for line in $(oc get pod -o=custom-columns=NODE:.spec.nodeName --no-headers --field-selector=status.phase!=Running -n node-gather)
+do
+    node=$(echo $line | awk -F ' ' '{print $1}')
```

Comment:
I think we can skip this line. $line should contain just the node name, so node==line

Reply:
fixed.